features/step_definitions/elements_steps.rb

```diff
@@ -26,10 +26,16 @@
   @test_site.home.wait_for_lots_of_links(0.1, count: 2)
   Capybara.using_wait_time(0.3) do
     # intentionally wait and pass nil to force this to cycle
-    expect(@test_site.home.wait_for_lots_of_links(nil, count: 198_108_14)).to be false
+    expect(@test_site.home.wait_for_lots_of_links(nil, count: 198_108_14))
+      .to be_falsey
   end
 end
 
-Then('I can wait a variable time for elements to disappear and pass specific parameters') do
-  expect(@test_site.home.wait_for_no_removing_links(0.1, text: 'wibble')).to be true
+Then(
```

Comment:
Same for this one, lets just change it to how it was and tackle it later. I think this format looks off